ext/opcache/Optimizer/zend_optimizer.c

```diff
@@ -511,6 +530,9 @@ int zend_optimizer_update_op2_const(zend_op_array *op_array,
 			if (opline->extended_value == ZEND_ASSIGN_OBJ) {
 				TO_STRING_NOWARN(val);
 				opline->op2.constant = zend_optimizer_add_literal(op_array, val);
+				(opline+1)->extended_value = alloc_cache_slots(op_array, 3);
+			} else if (opline->extended_value == ZEND_ASSIGN_STATIC_PROP) {
+				goto handle_static_prop;
 				(opline+1)->extended_value = alloc_cache_slots(op_array, 2);
```

Comment:
This line is dead code, due to goto above. Either needs to be swapped or removed.